### PR TITLE
fix: Pass rendered.headers to writeHead as object

### PR DESCRIPTION
**Pull request by timstrucken**
# Summary

Fixes headers not being passed from SvelteKit response object to Express Response object.

Fixes: #165

# Other Information

In order for the tests to successfully finish #173  has to be merged into main or this branch first.

### CODE REVIEW

**Comment by timstrucken** [comment]
@jthegedus what do you prefer: should I wait until #173 is merged into master and then merge master in this branch or do you want me to merge #173 in this branch directly? :)

**Review by jthegedus — APPROVED** [review]

**Comment by jthegedus** [comment]
Thanks!

**Pull request merged**
